Automattic/simplenote-ios
# Fixing Caret Repositioning in Authenticator

### Fix
In this PR we're fixing a glitch that prevents users from repositioning the Caret, in the Email field (Signup / SignIn).

@etoledom May I bother you with a real quick PR?
Thanks in advance!!

Closes #453

### Test
1. Perform a fresh install
2. Press over the **SignUp** button
3. Type any random text in the email field

- [ ] Verify you can, effectively, relocate the caret at will.

### Release
`RELEASE-NOTES.txt` was updated in 2246a67 with:

> Added markdown support
